magic.c

```diff
@@ -9,6 +9,7 @@ LGPL
 #include <lua.h>
 #include <lualib.h>
 #include <lauxlib.h>
+#include <assert.h>
```

Comment:
I think this should be within the `LUA_VERSION_NUM < 502` conditional below.
```c
#if LUA_VERSION_NUM < 502
#include <assert.h>
...
#else
```

Reply:
you mean like this
```c
#if LUA_VERSION_NUM < 502
#include <assert.h>
#endif
```